### PR TITLE
Aftershock Exosuit Power Up 2! MORE POWER

### PR DESCRIPTION
**Pull request by Drew4484**
#### Summary
Content "Adds a 'compact' ASRG power supply."

#### Purpose of change
Adds an ASRG module, this requires the entire PSU pocket.

#### Describe the solution
This module cannot be repaired, leaks radiation when damaged, and outweighs the entire frame with plastic plating. It provides endless but slow power and has a 10000 capacity battery.

#### Describe alternatives you've considered
None.

#### Testing
WIP

#### Additional context
Roughly based on the ASRG in #59479, this uses some of Aftershock's advanced materials to provide relatively lightweight shielding.

### CODE REVIEW

**Comment by anoobindisguise** [comment]
I don't have much stake in this as I'm not an aftershock person, but if it really is a 130w generator it's probably going to refill one charge per 8 seconds or so right? If I'm understanding the regenerate ammo function correctly.

**Comment by Drew4484** [comment]
> I don't have much stake in this as I'm not an aftershock person, but if it really is a 130w generator it's probably going to refill one charge per 8 seconds or so right? If I'm understanding the regenerate ammo function correctly.

Yeah, I was just testing different values now to get it where I wanted, but I think 8 is correct so I've set that and will test again.

File: data/mods/Aftershock/items/armor/exosuit/exosuit_modules.json [hunk 1]
```diff
@@ -109,6 +109,23 @@
       }
     ]
   },
+  {
+    "id": "exo_asrg",
+    "type": "TOOL_ARMOR",
+    "category": "exosuit",
+    "name": { "str": "exosuit ASRG" },
+    "description": "This is a prototype Compact Advanced Sterling Radioisotope Generator, intended to power an exosuit indefinitely.  The layered shielding saves space but is quite heavy, and while labeled 'Compact', this only just fits on the exosuit frame.  Due to the complexities of manufacturing, cannot be repaired, only replaced.",
```
Comment (TheShadowFerret) [inline data/mods/Aftershock/items/armor/exosuit/exosuit_modules.json:117]:
```suggestion
    "description": "This is a prototype Compact Advanced Sterling Radioisotope Generator, intended to power an exosuit indefinitely.  The layered shielding saves space but is quite heavy, and while labeled 'Compact', this only just fits on the exosuit frame.  Due to the complexities of manufacturing, it cannot be repaired, only replaced.",
```

File: data/mods/Aftershock/items/armor/exosuit/exosuit_modules.json [hunk 2]
```diff
@@ -109,6 +109,23 @@
...
+    "weight": "50 kg",
+    "volume": "25 L",
+    "material": [ "qt_steel", "plastic", "graphene_weave", "nanoprinted_alloy" ],
+    "symbol": "=",
+    "looks_like": "storage_battery",
+    "color": "dark_gray",
+    "ammo": "battery",
+    "flags": [ "FRAGILE", "CANT_WEAR", "WATERPROOF", "IS_UPS", "EXO_LARGE_GADGET", "LEAK_DAM", "RADIOACTIVE" ],
```
Comment (TheShadowFerret) [inline data/mods/Aftershock/items/armor/exosuit/exosuit_modules.json:125]:
```suggestion
    "flags": [ "FRAGILE", "CANT_WEAR", "WATERPROOF", "IS_UPS", "EXO_LARGE_GADGET", "LEAK_DAM", "RADIOACTIVE", "NO_REPAIR" ],
```
image
Cuz bug

**Review by John-Candlebury — APPROVED** [review]

**Pull request merged**